storybook/stories/API/chart/AreaChart.stories.tsx

```diff
@@ -13,18 +15,18 @@ export default {
 
 export const Simple = {
   render: (args: Record<string, any>) => {
-    const { data } = args;
     return (
       <ResponsiveContainer width="100%" height={400}>
-        <AreaChart data={data}>
-          <Area dataKey="pv" stroke="#2451B7" fill="url(#color)" />
+        <AreaChart {...args}>
```

Comment:
Nice, this does seem like a fix of the story as well 😅

Reply:
Ah yes I fixed it while I was in there